modular_skyrat/modules/cortical_borer/code/cortical_borer_abilities.dm

```diff
@@ -572,6 +573,7 @@
 	var/logging_text = "[key_name(cortical_owner)] went into [key_name(cortical_owner.human_host)] at [loc_name(human_turftwo)]"
 	cortical_owner.log_message(logging_text, LOG_GAME)
 	cortical_owner.human_host.log_message(logging_text, LOG_GAME)
+	ADD_TRAIT(cortical_owner, TRAIT_WEATHER_IMMUNE, "borer_in_host")
```

Comment:
```suggestion
	ADD_TRAIT(cortical_owner, TRAIT_WEATHER_IMMUNE, "borer_in_host") // BUBBER EDIT - WEATHER IMMUNE
```

Reply:
Coulda pinged me on dsicord or given me more than 3 days of time to do this but thanks